package.json

```diff
@@ -27,7 +29,10 @@
     "lint": "npm-run-all lint:*",
     "lint:addon": "npm run package && addons-linter ./addon.xpi -o text",
     "lint:js": "eslint addon",
-    "build": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js && web-ext build --source-dir=addon --overwrite-dest",
+    "build": "npm-run-all build:*",
+    "build:ga": "mkdir -p addon/build && cp $(node -e 'console.log(require.resolve(\"testpilot-ga\"))') addon/build/testpilot-ga.js",
+    "build:manifest": "mustache package.json manifest.json.tmpl > addon/manifest.json",
```

Comment:
Oh hey - for some reason I didn't remember that mustache had a CLI util. Would be good to keep this in my pocket for doing this in the future for an add-on not built with webpack

Reply:
I blatantly stole this solution from the fine folks on the Shield team. ❤️
But it seemed like a cheap solution to proxy those 4 configs I needed without need to add a bin/build-manifest.js file (or whatever).